core/drivers/crypto/hisilicon/qm/hisi_qm.c

```diff
@@ -0,0 +1,779 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
As suggested for the 4th commit I would prefer this file to be `core/drivers/crypto/hisilicon/hisi_qm.c`.